src/pair.h

```diff
@@ -156,6 +158,7 @@ class Pair : protected Pointers {
   virtual void compute_inner() {}
   virtual void compute_middle() {}
   virtual void compute_outer(int, int) {}
+  virtual double compute_atomic_energy(int, NeighList *) { return 0.0; }
```

Comment:
This function and the associated flag should be documented in https://docs.lammps.org/Modify_pair.html